bedrock/firefox/templates/firefox/nightly/whatsnew.html

```diff
@@ -40,7 +40,7 @@
   </header>
 
   <div class="mzp-l-content">
-    <div class="mzp-c-emphasis-box">
+    <div class="mzp-c-emphasis-box mzp-t-dark-override-color">
```

Comment:
Class names prefixed with `mzp` are reserved for [Protocol](https://protocol.mozilla.org/) components only, so we should avoid creating custom names like this in bedrock.